invenio_app_rdm/requests_ui/templates/semantic-ui/invenio_requests/macros/request_header.html

```diff
@@ -43,7 +43,7 @@ <h2 class="ui header">{{ request.title }}</h2>
       </div>
     {% endif %}
 
-    <div id="request-actions" class="flex align-items-start ml-auto mb-10">
```

Comment:
Before:
![Screenshot from 2022-09-05 14-48-57](https://user-images.githubusercontent.com/25476209/188454107-5384c260-0f9a-4a21-ab7f-10c9df2f440e.png)

After:
![Screenshot from 2022-09-05 14-49-08](https://user-images.githubusercontent.com/25476209/188454157-190b39a1-5afd-48fa-a761-d233cce128ca.png)

Reply:
was this discussed with @jennur ?